### PR TITLE
Adding Extension for Time Translation

### PR DESCRIPTION
**Pull request by nchapagain001**
This PR creates an extension method that can translates time from one unit to another. Example: minute -> seconds, Nanoseconds -> Milliseconds, etc.

### CODE REVIEW

File: src/VirtualClient/VirtualClient.Contracts.UnitTests/Parser/TextParserExtensionsTests.cs [hunk 1]
```diff
@@ -34,17 +35,62 @@ public void TextParserExtensionsTranslateByteUnitAsExpected(string orginalText,
...
+        [TestCase("1second", "1")]
+        [TestCase("2seconds", "2")]
+        public void TextParserExtensionsTranslateTimeUnitToSecondAsExpected(string originalText, string expectedOutput)
+        {
+            string result = TextParsingExtensions.TranslateToSecondUnit(originalText);
+            Assert.AreEqual(result, expectedOutput);
+        }
+
+        [Test]
+        [TestCase("60s", MetricUnit.Minutes, "1")]
```
Comment (yangpanMS) [inline src/VirtualClient/VirtualClient.Contracts.UnitTests/Parser/TextParserExtensionsTests.cs:84]: Great work! I would be more interested in checking more special cases like double(3.14), negative or zeros

File: src/VirtualClient/VirtualClient.Contracts.UnitTests/Parser/TextParserExtensionsTests.cs [hunk 2]
```diff
@@ -34,17 +35,62 @@ public void TextParserExtensionsTranslateByteUnitAsExpected(string orginalText,
...
+        [TestCase("1hrs", "3600")]
+        [TestCase("1hr", "3600")]
+        [TestCase("1hours", "3600")]
+        [TestCase("1hour", "3600")]
+        //
+        [TestCase("1m", "60")]
+        [TestCase("1minutes", "60")]
+        [TestCase("1minute", "60")]
+        //
+        [TestCase("1000ms", "1")]
```
Comment (yangpanMS) [inline src/VirtualClient/VirtualClient.Contracts.UnitTests/Parser/TextParserExtensionsTests.cs:62]: could we add upper case as well?

**Review by yangpanMS — APPROVED** [review]
Great work! check comments

File: src/VirtualClient/VirtualClient.Contracts.UnitTests/Parser/TextParserExtensionsTests.cs (hunk 2 above)
Comment (nchapagain001) [reply src/VirtualClient/VirtualClient.Contracts.UnitTests/Parser/TextParserExtensionsTests.cs:64]: Added

File: src/VirtualClient/VirtualClient.Contracts.UnitTests/Parser/TextParserExtensionsTests.cs (hunk 1 above)
Comment (nchapagain001) [reply src/VirtualClient/VirtualClient.Contracts.UnitTests/Parser/TextParserExtensionsTests.cs:101]: double and zero are supported. negative time is not.

File: src/VirtualClient/VirtualClient.Contracts.UnitTests/Parser/TextParserExtensionsTests.cs [hunk 3]
```diff
@@ -34,17 +35,84 @@ public void TextParserExtensionsTranslateByteUnitAsExpected(string orginalText,
...
+        public void TextParserExtensionsTranslateTimeUnitToSecondAsExpected(string originalText, string expectedOutput)
+        {
+            string result = TextParsingExtensions.TranslateToSecondUnit(originalText);
+            Assert.AreEqual(result, expectedOutput);
+        }
+
+        [Test]
+        [TestCase(" -2seconds ")]
+        [TestCase("-1minute")]
+        public void TextParserExtensionsDoesNotSupportNegativeTime(string originalText)
```
Comment (yangpanMS) [inline src/VirtualClient/VirtualClient.Contracts.UnitTests/Parser/TextParserExtensionsTests.cs:90]: Just curious, why we don't support negatives? They could be valid output from workload

**Review by yangpanMS — APPROVED** [review]

File: src/VirtualClient/VirtualClient.Contracts.UnitTests/Parser/TextParserExtensionsTests.cs (hunk 3 above)
Comment (nchapagain001) [reply src/VirtualClient/VirtualClient.Contracts.UnitTests/Parser/TextParserExtensionsTests.cs:90]: I have not experienced need for negative time in Virtual Client yet. 
Not supporting negative time (for now) to stay consistent with Byte function.

**Comment by nchapagain001** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Pull request merged**